UITests/AuthenticationTests.swift

```diff
@@ -3,6 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import Foundation
+import KIF
```

Comment:
What is `KIF`?

Reply:
Keep it Functional testing framework. We were globally importing it using a shared header, not sure why, as that isn't conventional swift style IMO. I removed the use of that header.